### PR TITLE
DOCS: Mention that libcrypto has helper functions for OSSL_PARAMs

### LINKED ISSUE #11165: OSSL_PARAM_REAL type
The documentation says:
```
       OSSL_PARAM_REAL
           The parameter data is a floating point value in native form.
```

It seems that the code has switches all over checking sizeof(double). I think double is the only thing we support. It's probably more clear that we say double there instead of "floating point".

### PR DESCRIPTION
**Pull request by levitte**
Fixes #11165

### CODE REVIEW

File: doc/man3/OSSL_PARAM.pod [hunk 1]
```diff
@@ -71,6 +71,13 @@ is NULL.  The usual full terminating template is:
 
 This can also be specified using L<OSSL_PARAM_END(3)>.
 
+=head2 Functional support
+
+Libcrypto offers a limited set of helper functions to handle
+B<OSSL_PARAM> items and arrays, please see L<OSSL_PARAM_get_int(3)>.
+Applications, providers, etc are free to extend or replace those as
```
Comment (t8m) [inline doc/man3/OSSL_PARAM.pod:78]: `etc.` ?

Comment (richsalz) [reply doc/man3/OSSL_PARAM.pod:78]: `, etc., `  Although it raises the question "what else?"  How about "Developers are free ..."

Also the intro comment says "Fixed" not "Fixes"

I will close my PR.

Comment (levitte) [reply doc/man3/OSSL_PARAM.pod:78]: I'll switch to your proposal, @richsalz.  Seems much simpler, yeah

**Review by t8m — APPROVED** [review]
LGTM Although I'm still unsure if something like: `The sending and receiving side of the B<OSSL_PARAM> of course have to agree on semantics and support of the individual data types.` shoudn't be added.

**Review by paulidale — APPROVED** [review]

**Comment by paulidale** [comment]
Merged to master.

**Pull request closed without merging**